migrations/schema.sql

```diff
@@ -510,25 +510,6 @@ END;
 $$;
 
 
---
--- Name: key_encrypt_secret(); Type: FUNCTION; Schema: pgsodium; Owner: -
---
-
-CREATE FUNCTION pgsodium.key_encrypt_secret() RETURNS trigger
```

Comment:
The `schema` test has been failing for a while, so I guess this just reflects the latest pgsodium version.

Reply:
https://github.com/michelp/pgsodium/blob/060e27fc93a53999d94c58170627a443f0eb8d0c/sql/pgsodium--3.1.0--3.1.1.sql#L178